readthedocs/core/views/hooks.py

```diff
@@ -225,9 +221,6 @@ def github_build(request):  # noqa: D205
                     branches
                 )
             projects = repo_projects | ssh_projects
-            # TODO remove _build_url call and replace with a 4xx response after
-            # Jan 31st.
-            DeprecatedGitHubWebhookNotification.for_project_users(projects)
```

Comment:
If we remove this from here, how is the idea to notify these users?

Reply:
Manually for now, we need this out soon. We can probably just keep this easy and ditch automating, and just do 2 or 3 scheduled emails instead.